### ISSUE TITLE
Tag new version?

### ISSUE THREAD

**Issue by tkelman** [one-more-minute/Lazy.jl#11]
I get an `ERROR: new not defined` in LazyList at Lazy.jl:45 on the tagged v0.8.3, win64 Julia 0.3.6. Lazy.jl master loads okay, not sure what the difference is.

Could you also push the tags here so they're visible on github? Makes keeping track of what version we're getting from a tag much easier. Turning on Travis would also be a very good idea.

**Comment by one-more-minute** [comment]
I can't reproduce this on OS X or Linux but strange things are happening on Windows.

The first time I tried `using Lazy` after a `Pkg.update()` I got some kind of dump/error before Julia quit, then after restarting I saw your issue. Checked out master, all fine. Checked out 0.8.3, issue was gone...

[Changeset since v0.8.3](https://github.com/one-more-minute/Lazy.jl/compare/v0.8.3...master) – I haven't even touched any of the LazyList code since then.
